boards/arm/nrf9160dk_nrf52840/dts/bindings/nordic,nrf9160dk-nrf52840-interface.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+# NOTE: This file is replicated in nrf9160dk_nrf9160 and nrf9160dk_nrf52840.
```

Comment:
I think it should be possible to use pre_dt_board.cmake from the nrf9160dk_nrf52840 board directory to manipulate DTS_ROOT. By adding the nrf9160dk_nrf9160 board directory to DTS_ROOT, this binding should be available without copy/pasting.

Reply:
I'm not sure how to do it properly, as I don't want the other files from `dts/` directory of one board to be available for the other. Only the two files: `nordic,nrf9160dk-nrf52840-interface.yaml` and `nordic,nrf9160dk-nrf52840-reset.yaml` are common for both these boards. Should I move these two files to another directory and add it to DTS_ROOT?

Reply:
Is there anything against having only nrf9160dk directory for both boards nrf9160dk_nrf9160 and nrf9160dk_nrf52840?